example/ipsec_crypto/odp_ipsec_misc.h

```diff
@@ -167,18 +167,18 @@ char *ipv4_addr_str(char *b, uint32_t addr)
 static inline
 int parse_ipv4_string(char *ipaddress, uint32_t *addr, uint32_t *mask)
 {
-	int b[4];
+	uint32_t b[4];
 	int qualifier = 32;
 	int converted;
 
 	if (strchr(ipaddress, '/')) {
-		converted = sscanf(ipaddress, "%d.%d.%d.%d/%d",
+		converted = sscanf(ipaddress, "%u.%u.%u.%u/%d",
```

Comment:
It would be more correct to use SCNu32 conversion specifier instead of 'u', to avoid problems in the very unlikely case that int is not 4 bytes. Or maybe just make the array element type unsigned int instead of uint32_t.